rpm/libnfnetlink.spec

```diff
@@ -0,0 +1,57 @@
+Name:           libnfnetlink
+Version:        1.0.2
+Release:        1
+Summary:        Netfilter netlink userspace library
+License:        GPLv2
+URL:            https://github.com/sailfishos/libnfnetlink
+Source0:        %{name}-%{version}.tar.bz2
+BuildRequires:	kernel-headers
+BuildRequires:  automake autoconf libtool pkgconfig
+BuildRequires:  make
+
+%description
+libnfnetlink is a userspace library that provides some low-level
+nfnetlink handling functions.  It is used as a foundation for other, netfilter
+subsystem specific libraries such as libnfnetlink_conntrack, libnfnetlink_log
+and libnfnetlink_queue.
+
+%package        devel
+Summary:        Netfilter netlink userspace library
+Requires:       %{name} = %{version}-%{release}
+Requires:       kernel-headers
+
+%description    devel
+libnfnetlink is a userspace library that provides some low-level
+nfnetlink handling functions.  It is used as a foundation for other, netfilter
+subsystem specific libraries such as libnfnetlink_conntrack, libnfnetlink_log
+and libnfnetlink_queue.
+
+%prep
+%autosetup -p1 -n %{name}-%{version}/upstream
+
+%build
+%autogen
+%configure --disable-static
+%make_build %{?_smp_mflags}
```

Comment:
This shouldn't be needed, the macro `%{?_smp_mflags}` is already used in %make_build.

Reply:
This is quite directly from Fedora without any modifications yet. If this, and the libnetfilter_conntrack works for the purpose I intended, I'll modify this and all other things you pointed out, thanks for that.